Refine the descriptions of -i and -o options in babel-node (#2161)
babel/website: docs/node.md

| ---------------------- | --------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
| `-e, --eval [script]`  |                             | Evaluate script                                                                                                                             |
| `-p, --print`          |                             | Evaluate script and print result                                                                                                            |

> | `-i, --ignore [regex]` | `node_modules`              | Ignore all files that match this regex when using the require hook. If both -i and -o are omitted, any file outside current working directory is ignored too. |

Reviewer: ```suggestion
| `-i, --ignore [regex]` | `node_modules`              | Ignore all files that match this regex when using the require hook. If both `-i` and `-o` are omitted, files outside the current working directory are also ignored. |
```

> | `-o, --only [regex]`   | current working directory   | Include all files that match this regex when using the require hook. If both -i and -o are omitted, any file outside current working directory is ignored too. |

Reviewer 2: Nit: `Only include the files` would be more explicit, since `Include all files` doesn't imply that what is not matched is excluded. Apart from that, the desc looks good :+1:

Author: I see. How about `Include only the files that match this regex when using the require hook and exclude the other files. If both -i and -o are omitted, any file outside current working directory is ignored too.` ?

(I'm not a native English speaker, so if you have a good alternative, please share it. Sorry in advance.)

Reviewer 2: I'm not a native speaker either :stuck_out_tongue: 

Your proposed description seems ok to me.

> | `-o, --only [regex]`   | current working directory   | Include only the files that match this regex when using the require hook and exclude the other files. If both -i and -o are omitted, any file outside current working directory is ignored too. |

Reviewer: ```suggestion
| `-o, --only [regex]`   | current working directory   | Only include (and exclude all other) files that match this regex when using the require hook. If both `-i` and `-o` are omitted, files outside the current working directory are also ignored. |
```
